infra/resources/_modules/cosmos/locals.tf

```diff
@@ -0,0 +1,13 @@
+locals {
+  wallet_cosmosdb_containers = [
+    # Each document represents a wallet instance
+    # The id_holder partition key is the tokenized identifier fo the user
+    {
+      name               = "wallet-instance-records"
+      partition_key_path = "/id_holder"
+      autoscale_settings = {
+        max_throughput = 2000
+      },
+    }
+  ]
```

Comment:
@silvicir

Reply:
I would keep the names `wallet-instances` for the container and `/userId` for the partition key. what do you think?